flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/CommonPythonBase.scala

```diff
@@ -136,6 +140,16 @@ trait CommonPythonBase {
     config.addAll(tableConfig.getConfiguration)
     config
   }
+
+  private def getRealEnvironment(env: StreamExecutionEnvironment): StreamExecutionEnvironment = {
```

Comment:
Should we also consider the `DummyExecutionEnvironment`?

Reply:
Currently it is unnecessary as the `DummyExecutionEnvironment` would not be passed to the `translateToPlan` methods.